cloud-integration.md

```diff
@@ -20,9 +20,10 @@ The Kubecost helm chart provides values which can enable or disable each cloud p
 
 Value | Default | Description
 --: | :--: | :--
-`.Values.kubecostModel.etlAssetReconciliationEnabled` | true | Enables Reconciliation processes and endpoints. Corresponds to `ETL_ASSET_RECONCILIATION_ENABLED` environment variable.
-`.Values.kubecostModel.etlCloudAssets` | true | Enables Cloud Assets processes and endpoints. Corresponds to `ETL_CLOUD_ASSETS_ENABLED` environment variable.
-
+`.Values.kubecostModel.etlAssetReconciliationEnabled` | true | Enables Reconciliation processes and endpoints. This Helm value corresponds to the `ETL_ASSET_RECONCILIATION_ENABLED` environment variable.
+`.Values.kubecostModel.etlCloudAssets` | true | Enables Cloud Assets processes and endpoints. This Helm value corresponds to the `ETL_CLOUD_ASSETS_ENABLED` environment variable.
+`.Values.kubecostModel.cloudAssetsExcludeProviderID` | false | Enabling this flag, pre-aggregates cloud assets at the service and user label level for higher performance build speed and improved asset query times, at the cost of no longer being able to see the cost of individual cloud assets. This Helm value corresponds to the `CLOUD_ASSETS_EXCLUDE_PROVIDER_ID` environment variable. Currently only available on AWS. Users should ensure that `eks:cluster-name` is enabled as described in the [AWS OOC documentation](https://github.com/kubecost/docs/blob/master/aws-out-of-cluster.md#step-2-tag-your-resources) to prevent double counting of EKS resources
```

Comment:
Can we more urgently label this an "experimental" feature? Sounds like there are still some concerns, and we'll potentially want to revoke this later. I just don't want to set expectations that this will be here forever, or that it's 100% bullet-proof.